.github/workflows/release.yml

```diff
@@ -0,0 +1,56 @@
+name: Build Release
+
+on:
+  release:
+    types: [ published ]
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: ref to build binary from
+        required: false
+  push:
+    branches:
+      - 'frank/build-release'
+
+jobs:
+  build-binary:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        cpu: [ "", "skylake" ]
```

Comment:
per the failing CI, probably easiest to remove "skylake" from here and just build on regular CPU.